XX.md

```diff
@@ -0,0 +1,1078 @@
+
+#### 1. Consortium Formation
+
+1. Relay operators publish Relay Identity Announcements (Kind 39100)
+2. Operators create Trust Acts (Kind 39101) toward relays they wish to collaborate with
+3. When mutual trust acts exist, relays begin sharing directory events
+4. Trust relationships can be inherited through the web of trust with appropriate confidence scoring
+
+#### 2. Directory Event Synchronization
+
+When a relay receives an event from a user, it:
+
+1. Validates the event signature and content
+2. If the event contains an `I` tag, verifies the identity proof-of-control signature
+3. Stores the event locally
+4. Updates key delegation usage tracking (if applicable)
+5. Identifies trusted consortium members based on current trust acts
+6. Determines replication targets based on event kind:
+   - **Directory Events**: Replicate to all trusted consortium members
+   - **Custom Kinds**: Replicate only to relays that have specified this kind in their `K` tag
+7. Replicates the event to appropriate trusted relays using Directory Event Replication Requests (Kind 39104)
+
+**Event Kind Matching:**
+- Check each trust act's `K` tag for the event's kind number
+- Only replicate to relays that have explicitly included the kind in their act
+- Directory events are always replicated regardless of `K` tag contents
+- Respect trust level when determining replication scope and frequency
+
+#### 3. Key Delegation Management
+
+Each relay in the consortium MUST implement key delegation limits and expiration:
+
+**Delegation Limits:**
+- Maximum 512 unused key delegations per relay identity
+- When limit is reached, oldest unused delegations are removed first
+- Delegations become "used" when referenced in any stored directory event
+
+**Expiration Policy:**
+- Key delegations expire after 30 days without usage
+- Expired delegations MUST be deleted during periodic cleanup
+- Usage timestamp is updated whenever a delegation is referenced
+
+**Cleanup Process:**
+- Run cleanup at least daily to remove expired delegations
+- Log delegation removals for audit purposes
+- Notify consortium members of delegation changes if configured
+
+#### 4. Conflict Resolution
+
+When conflicting directory events are received (same pubkey, same kind, different content):
+
+1. **Timestamp Priority**: Newer events replace older ones
+2. **Signature Chain Validation**: Verify the complete signature chain
+3. **Identity Verification**: Validate `I` tag signatures if present
+4. **Consensus Voting**: For disputed events, trusted relays vote on validity
+5. **Byzantine Fault Tolerance**: System remains functional with up to 1/3 malicious nodes
+
+#### 5. Trust Propagation
+
+Trust relationships can be inherited through the web of trust:
+
+1. If Relay A trusts Relay B with "high" trust
+2. And Relay B trusts Relay C with "medium" trust  
+3. Then Relay A may automatically trust Relay C with "low" trust
+4. Trust inheritance follows configurable policies and confidence thresholds
```

Comment:
I don't think it's this simple. What if you have a network split with multiple separate trusted in groups? How do you prevent a cabal from taking over the global namespace?